R/format_PEW_temp.R

```diff
@@ -0,0 +1,541 @@
+                  HatchDate_observed, HatchDate_min, HatchDate_max,
+                  BroodSize_observed, BroodSize_min, BroodSize_max,
+                  FledgeDate_observed, FledgeDate_min, FledgeDate_max,
+                  NumberFledged_observed, NumberFledged_min, NumberFledged_max,
+                  AvgEggMass, NumberEggs, AvgChickMass, NumberChicksMass,
+                  AvgTarsus, NumberChicksTarsus, OriginalTarsusMethod, ExperimentID)
+
+  return(Brood_data)
+
+}
+
+create_capture_PEW <- function(pew_data, Brood_data) {
+
+  Brood_data_sel <-
+    Brood_data %>%
+    dplyr::select(BreedingSeason, Species, PopID, BroodID, LocationID,
+                  HatchDate_observed)
+
+
+  Capture_data_temp <-
+    pew_data %>%
+    #### Rename variables
+    dplyr::rename(LocationID = NestboxID) %>%
+    dplyr::select(-DateRingingChicks) %>%
+    #### Create new variables
+    dplyr::mutate(#### ASSUMPTION: There is no captureID info, so I made this up
+                  CaptureID = paste(IndvID, LocationID,
+                                    paste0(lubridate::year(CaptureDate),
+                                           sprintf("%02d", lubridate::month(CaptureDate)),
+                                           sprintf("%02d", lubridate::day(CaptureDate))),
+                                    sep = "_"),
+      #### ASSUMPTION: Some times are strange, to simplify, round to whole hour
+      #### ? do they correspond only to the experimental observations? keep/remove?
+      CaptureTime = ifelse(is.na(ObservationTimeH),
+                           NA,
+                           paste(sprintf("%02d", round(ObservationTimeH, 0)),
+                                 "00", sep = ":")),
+      Sex_observed = ifelse(Sex == "Chick", NA_character_, substr(Sex, 1, 1)),
+      CaptureAlive = ifelse(Method != "Found dead", TRUE, FALSE),
+      #### ASSUMPTION: the same as CaptureAlive, as there is no additional data
+      ReleaseAlive = CaptureAlive,
+      CapturePopID = PopID,
+      CapturePlot = NA_character_,
+      ReleasePopID = ifelse(ReleaseAlive == TRUE, CapturePopID, NA_character_),
+      ReleasePlot = ifelse(ReleaseAlive == TRUE, CapturePlot, NA_character_),
+      #### ASSUMPTION: use NA, as there is no information
+      OriginalTarsusMethod = NA_character_,
+      WingLength = NA,
+      ExperimentID = dplyr::case_when(Experiment == "BSM - Griffioen et al. 2019 PeerJ" ~
+                                        "COHORT; PARENTAGE",
+                                      Experiment == "2h Temp D4" ~ "SURVIVAL",
+                                      Experiment == "2h Temp D4 + Handicaping Male: Griffioen et al. 2019 Front Ecol&Evol" ~
+                                        "SURVIVAL; PARENTAGE"),
+      #### NOT SURE ABOUT THIS
+      Age_observed = case_when(Age == "0" & Method == "ChickRinging" ~ 1,
+                               Age == "1" ~ 5,
+                               Age == ">=2" ~ 7,
```

Comment:
'>=2' means we don't know the age with certainty, so this should be 6 ("After 2nd year: full-grown bird hatched before last calendar year; year of hatching otherwise unknown.") rather than 7 ("3rd year: a bird hatched two calendar years before, and now in its third calendar year.") We have no way to tell if the bird is in its third calendar year or later.

Reply:
Ok, I have always found this way of noting the age confusing :)